libs/ml/tests/unit/layers/convolution_1d.cpp

```diff
@@ -44,12 +44,12 @@ TYPED_TEST(Convolution1DTest, set_input_and_evaluate_test)  // Use the class as
 
   SizeType const input_channels  = 3;
   SizeType const output_channels = 5;
-  SizeType const input_height    = 3;
+  SizeType const input_length    = 3;
```

Comment:
that renaming seems reasonable to me but it should comply with overall logic so i would rather wait for others opinions